sdk/client/src/listeners/network-listener/utils/utils.ts

```diff
@@ -321,17 +321,13 @@ function makeId(length: number) {
 	return result
 }
 
-export const createNetworkRequestId = (useOtelTraceId?: boolean) => {
+export const createNetworkRequestId = () => {
 	// Long enough to avoid collisions, not long enough to be unguessable
 	const requestId = makeId(10)
 
-	if (useOtelTraceId) {
-		const context = getActiveSpan()
-		const traceId = context?.spanContext().traceId
-		return [getNetworkSessionSecureID(), traceId ?? requestId]
-	}
-
-	return [getNetworkSessionSecureID(), requestId]
+	const context = getActiveSpan()
+	const traceId = context?.spanContext().traceId
+	return [getNetworkSessionSecureID(), traceId ?? requestId]
```

Comment:
should there always be an active span here? if not, should we start a span to avoid creating our own `id` format here? then we can remove the `requestId = makeId...` line too

Reply:
We discussed this on a call and decided to keep it around for now.